### PR TITLE
DOC: Add instructor notes

### PR DESCRIPTION
**Pull request by jhlegarreta**
Add instructor notes.

### CODE REVIEW

**Comment by jhlegarreta** [comment]
Folks, a few notes:
- I have not taught the course, so you may want to add/remove points depending on your experience or adjust depending on the contents. 
- By the same token, the `Technical tips and tricks`, `Common problems` sections may require especial attention.
- Given that I am not a native English speaker, grammar will require checking.

**Review by kaitj — COMMENTED** [review]
Thanks for putting this together! Looks good to me. Not sure if @josephmje has anything to add?

File: _extras/guide.md [hunk 1]
```diff
@@ -1,6 +1,121 @@
...
+{% include base_path.html %}
+
+relative path root: {{ relative_root_path }}
+
+## Instructor notes
+
+## Lesson motivation and learning objectives
+
+This lesson is designed to introduce learners to the analysis of diffusion
+Magnetic Resonance Imaging (dMRI) data using primarily Python. Although it is
```
Comment (josephmje) [inline _extras/guide.md:14]:
```suggestion
Magnetic Resonance Imaging (dMRI) data using Python. Although it is
```

File: _extras/guide.md [hunk 2]
```diff
@@ -1,6 +1,121 @@
...
+designed for learners who have no prior experience with diffusion MRI, basic
```
Comment (josephmje) [inline _extras/guide.md:15]:
```suggestion
designed for learners who have no prior experience with diffusion MRI, a basic
```

File: _extras/guide.md [hunk 3]
```diff
@@ -1,6 +1,121 @@
...
+Diffusion MRI is a feature-rich modality. Processing dMRI data involves
+numerous aspects, and can take a non-negligible amount of time. Being able
+to critically assess the obtained features requires practice, usually with
+heterogeneous data.
+
+Upon completion of the lesson, learners should be able to know the purpose and
+value of acquiring and analyzing diffusion data, as well as being able to
+process and analyze their own diffusion data. This lesson does not cover the
+details about the different dMRI data acquisition sequences. Similarly, this
+lesson does not cover quality assurance or check aspects of the processing,
```
Comment (josephmje) [inline _extras/guide.md:27]:
```suggestion
lesson does not cover quality assurance or checking aspects of the processing,
```

File: _extras/guide.md [hunk 4]
```diff
@@ -1,6 +1,121 @@
...
+neither does it focus on the importance of visualization for that purpose.
+
+## Lesson design
+
+#### [Introduction to Diffusion]({{ relative_root_path }}/{% link _episodes/01-intro-diffusion-and-dataset.md %})
+
+* If your workshop includes the [Introduction to MRI and BIDS](https://carpentries-incubator.github.io/SDC-BIDS-IntroMRI/) lesson,
+learners will have the necessary knowledge to better understand how diffusion
+images are generated and different from MRI data. Similarly, they will be
```
Comment (josephmje) [inline _extras/guide.md:36]:
```suggestion
images are generated and different from other MRI data acquisitions. Similarly, they will be
```

File: _extras/guide.md [hunk 5]
```diff
@@ -1,6 +1,121 @@
...
+sensitive to, and how diffusion is able to capture it.
+* Explain that there are other software and tools, such as [`3D Slicer`](https://www.slicer.org/),
+[`DSI Studio`](http://dsi-studio.labsolver.org/), [`ExploreDTI`](https://www.exploredti.com/),
+[`MRtrix`](https://www.mrtrix.org/), or [`TrackVis`](https://www.mrtrix.org/),
+to analyze or visualize diffusion MRI data.
+
+#### [Preprocessing dMRI data]({{ relative_root_path }}/{% link _episodes/02-diffusion-preprocessing.md %})
+
+* Pre-processing in dMRI depends on the available data (acquisition) and the
+quality of the data, so learners should be encouraged to be look at their data
```
Comment (josephmje) [inline _extras/guide.md:49]:
```suggestion
quality of the data, so learners should be encouraged to look at their data
```

File: _extras/guide.md [hunk 6]
```diff
@@ -1,6 +1,121 @@
...
+
+#### [Diffusion Tensor Imaging (DTI)]({{ relative_root_path }}/{% link _episodes/03-diffusion-tensor-imaging.md %})
+
+* Learners should be able to understand the use, relevance and limitations of
+the DTI model, both from the a clinical point of view, and a research setting.
+* Be sure to emphasize that even if the DTI model is unable to solve fiber
+crossings, the concepts and derived scalar maps are still relevant in
+practice.
+* Pay special attention to the visualization of the DTI model results in order
+to ensure that the anatomy and the diffusion are correctly registered.
```
Comment (josephmje) [inline _extras/guide.md:66]:
```suggestion
to ensure that the anatomical and the diffusion MR images are correctly registered.
```

File: _extras/guide.md [hunk 7]
```diff
@@ -1,6 +1,121 @@
...
+necessary to visualize the tractograms using different software.
+
+#### Concluding remarks
+
+* Try to provide the learners with a clear idea of where diffusion MRI sits
+among the rest of MRI or neuroimaging modalities.
+* Make sure that all learners are able to follow the lessons and acomodate
+the pace to the audience. Diffusion MRI being such a feature-rich imaging
+modality, it is important that the explained concepts are clear to the
+learners before passing on to a different aspect. Be prepared to be
```
Comment (josephmje) [inline _extras/guide.md:91]:
```suggestion
learners before continuing to the next aspect. Be prepared to be
```

**Review by josephmje — COMMENTED** [review]
Thank you @jhlegarreta ! I really liked this guide. I just made a few changes for sentence flow. We should make sure that the notes you mention are apparent in each lesson.

File: _extras/guide.md (hunk 1 above)
Comment (kaitj) [reply _extras/guide.md:14]: I think the "primarily Python" is on me. Most of the diffusion episodes are in Python except some of the steps in the preprocessing notebooks which rely on command-line (for eddy / topup) and registration with the T1w image. 

Not sure if there is a way around both of those, but can discuss more on Slack or in a separate issue/PR.

Comment (josephmje) [reply _extras/guide.md:14]: Good call. This suggestion can be ignored then.

File: _extras/guide.md (hunk 2 above)
Comment (jhlegarreta) [reply _extras/guide.md]: Done in ad60b89.

File: _extras/guide.md (hunk 3 above)
Comment (jhlegarreta) [reply _extras/guide.md]: Done in ad60b89.

File: _extras/guide.md (hunk 4 above)
Comment (jhlegarreta) [reply _extras/guide.md]: Done in ad60b89.

File: _extras/guide.md (hunk 5 above)
Comment (jhlegarreta) [reply _extras/guide.md]: Done in ad60b89.

File: _extras/guide.md (hunk 6 above)
Comment (jhlegarreta) [reply _extras/guide.md]: Done in ad60b89.

File: _extras/guide.md (hunk 7 above)
Comment (jhlegarreta) [reply _extras/guide.md]: Done in ad60b89.

**Comment by jhlegarreta** [comment]
@kaitj @josephmje this is ready to be merged, right?

**Review by kaitj — APPROVED** [review]
Looks good to me!

**Review by josephmje — APPROVED** [review]

**Pull request merged**